### PR TITLE
limit blocks

### PR DESCRIPTION
**Pull request by pbrocks**
Addressing this issue https://github.com/GatherPress/gatherpress/issues/206

### CODE REVIEW

File: includes/core/assets/js/event-deny-list.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Remove unwanted blocks from given post type.
+ */
+
+wp.domReady(function () {
+	wp.blocks.unregisterBlockType('gatherpress/events-list');
```
Comment (mauteri) [inline includes/core/assets/js/event-deny-list.js:6]: I would add `venue-information` to this one and leave `events-list` to be allowed on an event page.

File: includes/core/assets/js/venue-deny-list.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+];
+
+wp.domReady(function () {
+	Object.keys(disableBlocks).forEach(function (key) {
+		const blockName = disableBlocks[key];
+		if (blockName && wp.blocks.getBlockType(blockName) !== undefined) {
+			wp.blocks.unregisterBlockType(blockName);
+		}
+	});
+});
```
Comment (mauteri) [inline includes/core/assets/js/venue-deny-list.js:20]: JS files should be added to `entry` in webpack and handled with build script.

**Review by mauteri — CHANGES_REQUESTED** [review]
Left some notes

File: includes/core/assets/js/venue-deny-list.js (hunk 2 above)
Comment (pbrocks) [reply includes/core/assets/js/venue-deny-list.js:20]: @mauteri can you do that for me? IE, show me how?

**Comment by mauteri** [comment]
I did something like this in a theme framework I wrote:

PHP
`
 39         add_action( 'enqueue_block_editor_assets', [ $this, 'editor_enqueue_scripts' ] );


 43     public function editor_enqueue_scripts() : void {
 44         $asset = require_once CATALYST_CORE_PATH . '/assets/build/editor.asset.php';
 45 
 46         wp_enqueue_script(
 47             'catalyst-editor-js',
 48             CATALYST_CORE_URL . '/assets/build/editor.js',
 49             $asset['dependencies'],
 50             $asset['version'],
 51             true
 52         );
 53 
 54         $unregister_block_types = genesis_get_config( 'unregister-block-types' ) ?? [];
 55 
 56         wp_localize_script(
 57             'catalyst-editor-js',
 58             'catalystEditor',
 59             [
 60                 'unregisterBlockTypes' => $unregister_block_types,
 61             ]
 62         );
 63     }
`

JavaScript
`
  1 import domReady from '@wordpress/dom-ready';
  2 
  3 domReady( function() {
  4     for (let i = 0; i < catalystEditor.unregisterBlockTypes.length; i++) {
  5         wp.blocks.unregisterBlockType( catalystEditor.unregisterBlockTypes[i] );
  6     }
  7 });
`

File: includes/core/assets/js/venue-deny-list.js (hunk 2 above)
Comment (pbrocks) [reply includes/core/assets/js/venue-deny-list.js]: added to transpiler

**Review by mauteri — APPROVED** [review]

**Pull request merged**